pkg/discovery/summary_test.go

```diff
@@ -0,0 +1,43 @@
e available: 1 node(s) had taint {node.kubernetes.io/not-ready: }, that the pod didn't tolerate.","namespace":"kube-system"},{"name":"etcd-kind-control-plane","healthy":true,"ready":"Ready","namespace":"kube-system"},{"name":"kindnet-hknk7","healthy":true,"ready":"Ready","namespace":"kube-system"},{"name":"kindnet-qhzxn","healthy":true,"ready":"Ready","namespace":"kube-system"},{"name":"kindnet-x9fsq","healthy":true,"ready":"Ready","namespace":"kube-system"},{"name":"kube-apiserver-kind-control-plane","healthy":true,"ready":"Ready","namespace":"kube-system"},{"name":"kube-controller-manager-kind-control-plane","healthy":true,"ready":"Ready","namespace":"kube-system"},{"name":"kube-proxy-8jj5r","healthy":true,"ready":"Ready","namespace":"kube-system"},{"name":"kube-proxy-g79v6","healthy":true,"ready":"Ready","namespace":"kube-system"},{"name":"kube-proxy-lrtdp","healthy":true,"ready":"Ready","namespace":"kube-system"},{"name":"kube-scheduler-kind-control-plane","healthy":true,"ready":"Ready","namespace":"kube-system"},{"name":"coredns-74ff55c5b-ppjrs","healthy":false,"ready":"False","reason":"Unschedulable","message":"0/1 nodes are available: 1 node(s) had taint {node.kubernetes.io/not-ready: }, that the pod didn't tolerate.","namespace":"kube-system"},{"name":"coredns-74ff55c5b-s89p7","healthy":false,"ready":"False","reason":"Unschedulable","message":"0/1 nodes are available: 1 node(s) had taint {node.kubernetes.io/not-ready: }, that the pod didn't tolerate.","namespace":"kube-system"},{"name":"local-path-provisioner-78776bfc44-66xwg","healthy":true,"ready":"Ready","namespace":"local-path-storage"},{"name":"sonobuoy","healthy":true,"ready":"Ready","namespace":"sonobuoy"},{"name":"sonobuoy-e2e-job-1d0e9c62fc114e08","healthy":true,"ready":"Ready","namespace":"sonobuoy"},{"name":"sonobuoy-systemd-logs-daemon-set-7a10f4effeb44d32-7n2cj","healthy":true,"ready":"Ready","namespace":"sonobuoy"},{"name":"sonobuoy-systemd-logs-daemon-set-7a10f4effeb44d32-jrn9s","healthy":true,"ready":"Ready","namespace":"sonobuoy"},{"name":"sonobuoy-systemd-logs-daemon-set-7a10f4effeb44d32-zqfjc","healthy":true,"ready":"Ready","namespace":"sonobuoy"}]},"api_version":"v1.20.0","error_summary":{"Errors":{"podlogs/kube-system/kube-apiserver-kind-control-plane/logs/kube-apiserver.txt":4,"podlogs/kube-system/kube-controller-manager-kind-control-plane/logs/kube-controller-manager.txt":44,"podlogs/kube-system/kube-scheduler-kind-control-plane/logs/kube-scheduler.txt":43},"Warnings":{"podlogs/kube-system/kube-apiserver-kind-control-plane/logs/kube-apiserver.txt":11,"podlogs/kube-system/kube-controller-manager-kind-control-plane/logs/kube-controller-manager.txt":13,"podlogs/kube-system/kube-proxy-8jj5r/logs/kube-proxy.txt":1,"podlogs/kube-system/kube-proxy-g79v6/logs/kube-proxy.txt":1,"podlogs/kube-system/kube-proxy-lrtdp/logs/kube-proxy.txt":1,"podlogs/kube-system/kube-scheduler-kind-control-plane/logs/kube-scheduler.txt":4,"podlogs/sonobuoy/sonobuoy/logs/kube-sonobuoy.txt":2}}}`)
```

Comment:
When you have this much data you really want to try and do something besides hardcode it like this. Its hard to consume (you can't easily cat the data or pipe it on the CLI), it makes your IDE a bit odd since its such a long line, and when/if changes are made it can be hard to update.

So thats what the "golden file" tests are for. That way the long, unwieldy data goes in its own file and if we want to update it we just run the test with the `-update` flag.

It can be odd to implement the first time so I'm happy to help if you're not sure how to apply that pattern (we use that pattern elsewhere so you can look at how its done though and dont have to start from scratch).